migrations/lib/uploads/cli.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require_relative "../migrations"
+require_relative "./settings"
+require_relative "./fixer"
+require_relative "./uploader"
+require_relative "./optimizer"
+
+module Migrations
+  load_rails_environment
+
+  load_gemfiles("common")
+  configure_zeitwerk("lib/common")
+
+  module Uploads
+    class CLI < Thor
```

Comment:
I think it makes sense to keep CLI code in `bin/process_uploads`. It's not really reusable code that you'd expect to find in `lib`. It doesn't help with testability, either. But maybe I'm missing something, what are your arguments for putting it into `lib`?

Reply:
This will be great for if we ever want to compose the CLIs into a single "migration" CLI, making each a sub command will be relatively straightforward.

> It doesn't help with testability, either.
Could you explain this further?